CMakeLists.txt

```diff
@@ -180,6 +180,7 @@ find_package(PythonInterp 3.6 REQUIRED)
 # When updating the version here, also update that used in the merge request
 # config
 find_package(ClangTools 16 COMPONENTS clang-format clang-tidy)
```

Comment:
is it intentional that we're still trying to find clang-tidy at version 16? Will it print a message that it wasn't found?

Reply:
oops I meant to take that bit out